Dockerfile

```diff
@@ -9,16 +8,14 @@ RUN apt update && apt upgrade -y
 RUN apt update && apt install -y gnupg wget lsb-release vim tmux libglvnd0 libglvnd-dev x11-apps mesa-utils python3-pip htop
```

Comment:
Only install tmux, vim, htop, x11-apps, mesa-utils and python3-pip
Delete line 7